airbyte-config/config-models/src/main/resources/types/StandardSyncSummary.yaml

```diff
@@ -28,6 +34,22 @@ properties:
     type: integer
   endTime:
     type: integer
+  replicationStartTime:
+    type: integer
+  replicationEndTime:
+    type: integer
+  normalizationStartTime:
```

Comment:
I think we already have normalization start and end time on NormalizationSyncSummary on the SyncOutput object

Reply:
Updated via https://github.com/airbytehq/airbyte/pull/17643/commits/0c6f93ebcfb716b674f1f6b4e7ac290ccda80ba9